apps/studio/components/interfaces/Settings/Database/ConnectionPooling/ConnectionPooling.tsx

```diff
@@ -256,10 +243,20 @@ export const ConnectionPooling = () => {
                                 label="Transaction"
                                 value="transaction"
                               >
-                                Transaction
+                                <p>Transaction mode</p>
+                                <p className="text-xs text-foreground-lighter">
+                                  Connection is assigned to the client for the duration of a
+                                  transaction. Some session-based Postgres features such as prepared
+                                  statements are not available with this option.
+                                </p>
                               </Listbox.Option>
                               <Listbox.Option key="session" label="Session" value="session">
-                                Session
+                                <p>Session mode</p>
+                                <p className="text-xs text-foreground-lighter">
+                                  When a new client connects, a connection is assigned to the
+                                  clilent until it disconnects. All Postgres features can be used
```

Comment:
typo "clilent" here. It should be changed to "client" instead.

Reply:
```suggestion
                                  client until it disconnects. All Postgres features can be used
```

Reply:
thank you so much @vhpx! 😄 🙏